toolbox.sh

```diff
@@ -0,0 +1,240 @@
+function start() {
+    # Check that the plugin is present in the target folder
+    if ! [[ -f $ECOCODE_JAVA_PLUGIN_JAR ]]; then
+        error "Cannot find ecoCode plugin in target directory" && return 1
+    fi
+    info "Starting Docker containers"
+    debug "docker compose --env-file $CURRENT_PATH/docker.env -f $CURRENT_PATH/docker-compose.yml start"
+    docker compose --env-file "$CURRENT_PATH/docker.env" -f "$CURRENT_PATH/docker-compose.yml" start
+    return 0
+}
+
+# @description Stopping Docker containers.
+# @noargs
+# @exitcode 0 If successful.
+function stop() {
+    info "Stopping Docker containers"
+    debug "docker compose --env-file $CURRENT_PATH/docker.env -f $CURRENT_PATH/docker-compose.yml stop"
+    docker compose --env-file "$CURRENT_PATH/docker.env" -f "$CURRENT_PATH/docker-compose.yml" stop
+    return 0
+}
+
+# @description Stop and remove containers, networks and volumes.
+# @noargs
+# @exitcode 0 If successful.
+function clean() {
+    info "Remove Docker containers, networks and volumes"
+    debug "docker compose --env-file $CURRENT_PATH/docker.env -f $CURRENT_PATH/docker-compose.yml down --volumes"
+    docker compose --env-file "$CURRENT_PATH/docker.env" -f "$CURRENT_PATH/docker-compose.yml" down --volumes
+    return 0
+}
+
+# @description Display Docker container logs.
+# @noargs
+# @exitcode 0 If successful.
+function display_logs() {
+    info "Display Docker container logs"
+    debug "docker compose --env-file $CURRENT_PATH/docker.env -f $CURRENT_PATH/docker-compose.yml logs -f"
+    docker compose --env-file "$CURRENT_PATH/docker.env" -f "$CURRENT_PATH/docker-compose.yml" logs -f
+    return 0
+}
+
+# @description Check if docker is correctly installed.
+# @noargs
+# @exitcode 0 If successful.
+# @exitcode 1 If docker is not installed.
+# @exitcode 2 If the docker compose module is not installed.
+# @exitcode 3 If the minimum version is not installed.
+function check_env_docker() {
+    # Check if docker is installed
+    ! [[ -x "$(command -v docker)" ]] && error "Please install docker" && return 1
+    # Check if the docker compose module is installed
+    if ! [[ -x "$(command -v docker compose|tail -n1|grep compose)" ]]; then
+        error "Please install docker compose module" && return 2
+    fi
+    # Check docker compose version
+    local dc_version_major dc_version_current
+    dc_version_major=$(docker compose  version  --short | cut -d '.' -f 1)
+    dc_version_current=$(docker compose  version  --short)
+    if [[ $dc_version_major -lt 2 ]]; then
+      error "$dc_version_current is not a supported docker compose version, please upgrade to the minimum supported version: 2.0"
+      return 3
+    fi
+    return 0
+}
+
+# @description Check local environment.
+# @noargs
+# @exitcode 0 If successful.
+# @exitcode 1 If docker is not correctly installed.
+# @exitcode 2 If java is not installed.
+# @exitcode 3 If maven is not installed.
+function check_env() {
```

Comment:
WARNING ... there is already a documentation and a script to check environment requirements : please check documentation here https://github.com/green-code-initiative/ecoCode-common/blob/main/doc/starter-pack.md#method-1---automatic-check
maybe make a reference on this script (one for unix / MAc users, and one for windows users)